### PR TITLE
Fixed Filebeat ECS dashboards

### PR DESCRIPTION
**Pull request by EthanStrider**

### CODE REVIEW

File: filebeat/module/mysql/_meta/kibana/7/dashboard/Filebeat-mysql.json [hunk 1]
```diff
@@ -28,7 +28,7 @@
                             "id": "1", 
                             "params": {
                                 "customLabel": "Query time", 
-                                "field": "mysql.slowlog.query_time.sec"
+                                "field": "event.duration"
```
Comment (ruflin) [inline filebeat/module/mysql/_meta/kibana/7/dashboard/Filebeat-mysql.json]: This one changes the unit, so it's not a 1-1 mapping. We should follow up to see if the dashboard still looks as expected.

File: filebeat/module/nginx/_meta/kibana/7/dashboard/ml-nginx-access-remote-ip-count-explorer.json [hunk 2]
```diff
@@ -40,7 +40,7 @@
                             "enabled": true, 
                             "id": "3", 
                             "params": {
-                                "field": "nginx.access.remote_ip", 
+                                "field": "source.address",
```
Comment (ruflin) [inline filebeat/module/nginx/_meta/kibana/7/dashboard/ml-nginx-access-remote-ip-count-explorer.json]: I can't find this on in our ecs migration list. Seems like we missed this one.

**Comment by ruflin** [comment]
I wonder why these didn't get properly renamed. Perhaps we touched these after the rename?

**Comment by ruflin** [comment]
@EthanStrider Thanks for the fix and digging into this. Will forward port it to master.

**Comment by EthanStrider** [comment]
Thanks!
Related: https://github.com/elastic/beats/pull/11512

**Comment by ruflin** [comment]
Yep, also merged this one and will backport to 7.0 now. Let me know if you find any other broken dashboards.

File: libbeat/docs/field-name-changes.asciidoc [hunk 3]
```diff
@@ -2,6 +2,7 @@
 [frame="topbot",options="header"]
 |======================
 |Old Field|New Field
+|`auditd.message_type`            |`event.type`
```
Comment (EthanStrider) [inline libbeat/docs/field-name-changes.asciidoc]: @webmat Do we need to revert this change?

File: libbeat/docs/field-name-changes.asciidoc [hunk 4]
```diff
@@ -12,7 +13,6 @@
 |`docker.container.image`            |`container.image.name`
 |`docker.container.labels`            |`container.labels`
 |`docker.container.name`            |`container.name`
-|`event.type`            |`auditd.message_type`
```
Comment (webmat) [inline libbeat/docs/field-name-changes.asciidoc]: This needs to be put back as it was. `event.type` is reserved and should not be used in 7.0

File: filebeat/module/nginx/_meta/kibana/7/dashboard/ml-nginx-access-remote-ip-count-explorer.json (hunk 2 above)
Comment (webmat) [reply filebeat/module/nginx/_meta/kibana/7/dashboard/ml-nginx-access-remote-ip-count-explorer.json]: This was the main reason for https://github.com/elastic/beats/pull/11512. You'll see it when it's backported ;-)

File: libbeat/docs/field-name-changes.asciidoc (hunk 3 above)
Comment (webmat) [reply libbeat/docs/field-name-changes.asciidoc]: Yes it should :-)
